extensions/amp-viewer-integration/0.1/findtext.js

```diff
@@ -347,6 +347,9 @@ export class TextScanner {
       // Accessing display of computed styles does not force layout/reflow
       // unless media queries that require relayout are used.
       // https://jsfiddle.net/7c7rq2ot/
+      //
+      // Note this does not eliminate all hidden elements (e.g. visibility:hidden).
+      // TODO(yunabe): Support more hidden element patterns if necessary.
       const display = computedStyle(window, node).display;
```

Comment:
1. use `ampdoc.win`
2. destructure: `const {display} = computedStyle(ampdoc.win);`

Reply:
Rewrote APIs in this file to pass around `win` object.
Also, fixed all lint errors.

PTAL?